lib/rust/flexer/tests/flexer_generated_api_test.rs

```diff
@@ -9,16 +9,25 @@
 //! space       = ' ';
```

Comment:
Do we need to keep the generated file in the codebasE? I would not do it. If we have tests which generate the code and we can inspect it working there, letsn to keep it here.